docs/architecture/platform-versions.md

```diff
@@ -88,6 +88,9 @@ History:
   discovered that we'd been mistaken in thinking since 2018 that its
   WebView browser got updated independently; in fact it's pinned at a
   version a couple of years older than any other browser we support.
+* We dropped iOS 10 support in 2020-10. It was <0.3% of iOS users
```

Comment:
nit: just "was 0.3%" -- in the message cited, that figure is rounded and the exact figure might have been anything from 0.25% to less than 0.35%